definitions/divorce/json/CaseEvent/CaseEvent.json

```diff
@@ -2686,6 +2686,8 @@
     "DisplayOrder": 3,
     "PreConditionState(s)": "DNisRefused;AwaitingHWFDecision;Submitted;Issued;Rejected;PendingRejection;solicitorAwaitingPaymentConfirmation;AosAwaiting;AosStarted;AosSubmittedAwaitingAnswer;AwaitingDecreeNisi;AosOverdue;AwaitingReissue;AosCompleted;AosAwaitingSol;AosPreSubmit;AosDrafted;AwaitingService",
     "PostConditionState": "AmendPetition",
+    "CallBackURLAboutToSubmitEvent": "${CCD_DEF_COS_URL}/solicitor-amend-petition-dn-rejection",
```

Comment:
Are you sure this Event can be updated like this and this can go on prod already? 

Is it "toggled" off on authorisations?